test/evergreen.yml

```diff
@@ -418,9 +418,9 @@ functions:
           # Compiling with CMake generated Ninja file.
           cd cmake_build
           if [ "${cmake_generator|Ninja}" = "Ninja" ]; then
-            ninja -j ${num_jobs} 2>&1
+            ninja 2>&1
           else
-            make -j ${num_jobs} 2>&1
+            make 2>&1
```

Comment:
Did we drop the `num_jobs` from `make` because of the later tasks that set it to 1? `make` doesn't automatically parallelise like `ninja` does, so this second branch is likely to be much slower now. If the problem is that variables for the compile and the execution get munged together, can we split it like `num_compile_jobs` vs `num_test_jobs` (or whatever naming works better)?

Reply:
I returned back -j for make, please take a look